### PR TITLE
update bc4a75 to aria 1.2

### LINKED ISSUE #1900: bc4a75: Pass example 6 no longer passing in ARIA 1.2
https://act-rules.github.io/rules/bc4a75

Pass example 6:
```
<div role="list">
	<span role="listitem">Item 1</span>
	<div role="group">
		<span role="listitem">Item 2</span>
		<span role="listitem">Item 3</span>
	</div>
</div>
```

ARIA 1.1 allows a group child for `list`: https://www.w3.org/TR/wai-aria-1.1/#list
ARIA 1.2 DOES NOT allow a group child: https://www.w3.org/TR/wai-aria-1.2/#list

### PR DESCRIPTION
**Pull request by carlosapaduarte**
In addition to updating references from 1.1 to 1.2, this update also includes

- removing the combobox exception from the applicability 
- updating examples accordingly to the updated owning requirements
- moving list > group > listitem example from passed to failed 

Closes issue(s):

- closes #1900

Need for Call for Review:
This will require a 2 weeks Call for Review

---

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/aria-required-owned-element-bc4a75.md [hunk 1]
```diff
@@ -52,17 +49,15 @@ If the [explicit semantic role][] on the target element is incorrectly used, and
...
-
-The combobox role is excluded from this rule, because the design pattern for it as described in ARIA 1.1 has proven problematic. The combobox will be significantly different for ARIA 1.2, where it does not have [required owned elements][].
+The applicability of this rule is limited to the [WAI-ARIA 1.2 Recommendation][wai-aria 1.2] roles. The [WAI-ARIA Graphics Module][] does not include any [required owned elements][]. The [Digital Publishing WAI-ARIA Module (DPUB ARIA) 1.0][dpub 1.0] only has three roles with [required owned elements][] (`doc-bibliography`, `doc-endnotes` and `doc-glossary`); they have issues with their use of role inheritance, and all of them have no [required owned elements][] in the [Digital Publishing WAI-ARIA Module (DPUB ARIA) 1.1][dpub 1.1] editor's draft.
```
Comment (WilcoFiers) [inline _rules/aria-required-owned-element-bc4a75.md]: Should we simplify?

```suggestion
The applicability of this rule is limited to the [WAI-ARIA 1.2 Recommendation][wai-aria 1.2] roles. The [WAI-ARIA Graphics Module][] and [Digital Publishing WAI-ARIA Module (DPUB ARIA) 1.1 (Editors draft)][dpub 1.1] do not include any [required owned elements][].
```

**Review by WilcoFiers — DISMISSED** [review]
Possible simplification.

**Review by WilcoFiers — APPROVED** [review]

File: _rules/aria-required-owned-element-bc4a75.md [hunk 2]
```diff
@@ -26,16 +26,13 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any [HTML or SVG element][] that is [included in the accessibility tree][] and has a [WAI-ARIA 1.1][] [explicit semantic role][] with [required owned elements][], except if one or more of the following is true:
-
-- the element is a [semantic][semantic role] `combobox`; or
-- the element has the `aria-busy` [attribute value][] of `true`, or has an [ancestor][] in the accessibility tree with this [attribute value][].
+This rule applies to any [HTML or SVG element][] that is [included in the accessibility tree][] and has a [WAI-ARIA 1.2][] [explicit semantic role][] with [required owned elements][], except if the element has the `aria-busy` [attribute value][] of `true`, or has an [ancestor][] in the accessibility tree with this [attribute value][].
```
Comment (Jym77) [inline _rules/aria-required-owned-element-bc4a75.md]:
```suggestion
This rule applies to any [HTML or SVG element][] that is [included in the accessibility tree][] and has a [WAI-ARIA 1.2][] [explicit semantic role][] with [required owned elements][], except if the element has an [inclusive ancestor][] in the accessibility tree with an `aria-busy` [attribute value][] of `true`.
```

**Review by Jym77 — DISMISSED** [review]
Streamlining Applicability.

**Review by daniel-montalvo — APPROVED** [review]

**Review by Jym77 — APPROVED** [review]

**Comment by carlosapaduarte** [comment]
Merging

**Pull request merged**